Automattic/jetpack
# Settings UI: update VaultPress notice markup when it's moved into notices area

Note: this must only be merged after this dops PR is merged https://github.com/Automattic/dops-components/pull/83

#### Changes proposed in this Pull Request:
- updates the markup to match the new one in the dops notice

#### Testing instructions:
* make sure there is no padding that separates the button from the edge of the notice
